susemanager-utils/susemanager-sls/src/modules/mgr_caasp_manager.py

```diff
@@ -175,19 +196,115 @@ def add_node(skuba_cluster_path,
     return ret
 
 
+def _upgrade_cluster_plan(skuba_cluster_path,
```

Comment:
@ereslibre : for this file only and related to `skuba upgrade`: can you please review?
It should be as per your comment in https://github.com/uyuni-project/uyuni/pull/2128#discussion_r414368146